auth_pysaml/models/__init__.py

```diff
@@ -0,0 +1,8 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from . import auth_saml
```

Comment:
Name files according to model names:

```suggestion
from . import auth_saml_provider
```